crypto.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/hashgraph/hedera-protobufs-go/services"
 	"github.com/pkg/errors"
 	"golang.org/x/crypto/pbkdf2"
+	protobuf "google.golang.org/protobuf/proto"
 )
 
 const _Ed25519PrivateKeyPrefix = "302e020100300506032b657004220420"
```

Comment:
Note for the future: This is not actually correct.

Reply:
We have the same in JS

Reply:
I guess so but for example `30740201010420` would still be a valid prefix if you integrate the key data, curve data and etc. after it. Thats why I put in this notice.